### PR TITLE
fix(schematics): get rid of schematics-utilities

### PR DESCRIPTION
**Pull request by L2jLiga**
Schematics-utilities package was introduced in 6d74f4f022586794a80281082c93e8252fb5d72a
Usages of this package was removed in 372f848448fd384d04d50d1314736c34e81bd832

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/spectator/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #292 

## What is the new behavior?
schematics-utilities removed from dependencies list

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/spectator/schematics/package.json [hunk 1]
```diff
@@ -3,9 +3,8 @@
   "version": "0.0.0",
   "schematics": "./src/collection.json",
   "peerDependencies": {
-    "@angular-devkit/core": "^7.3.1",
-    "@angular-devkit/schematics": "^7.3.1",
-    "schematics-utilities": "^1.1.1",
-    "typescript": "~3.1.1"
+    "@angular-devkit/core": ">= 8.0.0",
```
Comment (NetanelBasal) [inline projects/spectator/schematics/package.json:6]: Why this change?

Comment (L2jLiga) [reply projects/spectator/schematics/package.json:6]: To be consist with https://github.com/ngneat/spectator/blob/master/projects/spectator/package.json

**Comment by NetanelBasal** [comment]
Please remove from all places https://github.com/ngneat/spectator/search?q=schematics-utilities&unscoped_q=schematics-utilities

**Comment by L2jLiga** [comment]
PR updated, all usages removed
